crates/katana/core/src/service/messaging/starknet.rs

```diff
@@ -104,7 +103,12 @@ impl StarknetMessaging {
             }
         }
 
-        Ok(block_to_events)
+        // Convert Hashmap into Vec
```

Comment:
Would be better to have the comment saying why it's important to keep order. 👍

Reply:
Could you please remind me why it is important to keep the order of the events?

Reply:
If blocks are not processed in order, then the messages are not executed in order which can cause execution errors.
This is why we do want to be sure the order is conserved. 👍